user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Configuring_the_default_upgrade_options.md

```diff
@@ -25,6 +25,6 @@ To do so:
    - *Failover policy*: Determines in which order the DMA in a Failover are upgraded.
 
      > [!NOTE]
-     > We highly recommend that you set this to *Upgrade main and backup Agent simultaneously*.
+     > We highly recommend that you set this to *Upgrade main and backup Agent simultaneously*, which is also the *default policy*.
```

Comment:
This change does not make sense. "Upgrade main and backup Agent simultaneously" is only the default policy if you configure it to be the default policy, which is exactly what the user does here. I'd propose that instead we update line 25-28 as follows to make this clear: 

```
- *Failover policy*: This option allows you to select the default policy for upgrading a Failover setup. This determines in which order the DMA in a Failover setup are upgraded by default.

     > [!IMPORTANT]
     > We highly recommend that you set the default Failover policy to *Upgrade main and backup Agent simultaneously*.
```